src/configure/utilities/webAppDotNetCoreVersionConverter.ts

```diff
@@ -0,0 +1,3 @@
+export async function webAppDotNetCoreRuntimeConverter(dotnetcoreVersion: string) {
+    return "DOTNETCORE|" + dotnetcoreVersion.replace("netcoreapp", "");
```

Comment:
this need not be an asset. you can use a string manipulation mustache helper to achieve this.

Reply:
using mustache helper instead